src/tests/ftest/container/snapshot.py

```diff
@@ -209,16 +209,22 @@ def test_snapshot_negativecases(self):
         thedata = "--->>>Happy Daos Snapshot-Create Negative Testing " + \
                   "<<<---" + rand_str(random.randint(1, data_size))
         try:
-            obj, epoch = self.container.write_an_obj(
-                thedata, len(thedata)+1, dkey, akey, obj_cls=obj_cls)
+            tx_handle = self.container.get_new_tx()
+            obj = self.container.write_an_obj(thedata,
+                                              len(thedata)+1,
+                                              dkey,
+                                              akey,
+                                              obj_cls=obj_cls,
+                                              txn=tx_handle)
+            self.container.commit_tx(tx_handle)
```

Comment:
I am ok to keep the new_tx + commit here, but that's really not required.

Reply:
Currently Snapshot.create() Python function is not returning anything and further test is trying to display it for information purpose. So I use the tx_handle to avoid any further changes in the test.

        self.log.info("==(1)Container epoch= %s", tx_handle)
        self.log.info("     snapshot.epoch= %s", snapshot.epoch)